manifests/o/Oracle/JDK-LTS/11.0.12.0/Oracle.JDK-LTS.installer.yaml

```diff
@@ -13,7 +13,7 @@ Installers:
   InstallerType: exe
   Scope: machine
   InstallerUrl: https://download.oracle.com/otn/java/jdk/11.0.12+8/f411702ca7704a54a79ead0c2e0942a3/jdk-11.0.12_windows-x64_bin.exe
```

Comment:
```
  InstallerUrl: https://download.oracle.com/otn/java/jdk/11.0.12+8/f411702ca7704a54a79ead0c2e0942a3/jdk-11.0.12_windows-x64_bin.exe
```
This link redirects to a login page which is why the pipelines are erroring out.